stingray/bispectra.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+from scipy.linalg import hankel
+import scipy.fftpack
+import stingray.lightcurve as lightcurve
+import stingray.utils as utils
+from stingray.utils import simon
+__all__ = ["bispectra"]
+
+
+class bispectra(object):
+
+  def __init__(self, lc=None,m=None, nfft=None):
+    """
+   
+   
+    Parameters 
+    --------
```

Comment:
There's lot to be done on code (boundary conditions too ) working on that , Just made rough initial commit so we can parallelize